### ISSUE TITLE
SV Industrial Multitool - Repair Lasers do not give any visual appearance while firing

### ISSUE THREAD

**Issue by StHotPants** [RavienGaming/Reforged-Eden-2#92]
### Reforged Eden 2 Build Number

RE2 Build 8

### Game mode

Survival

### Tested in vanilla

Not tested in vanilla or not applicable

### Description of issue

While performing the Repair option using a SV Industrial Multitool there is no visual appearance of a "repair laser" both in first person and third person view. The damaged blocks do repair and occasionally you will see a light blue "spark" flash on the screen for an instant, but there is no other visual indicator it is working.

**Comment by StHotPants** [comment]
Also, regardless of how many Industrial Multitools you have it only repairs one block at a time without regard to the multitool laser layout.

**Comment by VermillionX** [comment]
Yep. Doesn't look like it can be fixed. It's faulty Empyrion code.
